debian/changelog

```diff
@@ -1,3 +1,10 @@
+flatpak (1.3.3-1endless2) unstable; urgency=medium
+
+  * Add libsecret build dependency
+  * https://phabricator.endlessm.com/T26715
+
+ -- Damián Nohales <damian@endlessm.com>  Tue, 21 May 2019 16:39:32 -0300
```

Comment:
You don't actually need to add a new changelog entry unless you want to tag a new version. Jenkins will just make up a version based on the current tags without this.